src/lib/datepicker/datepicker.spec.ts

```diff
@@ -1736,3 +1770,26 @@ class DatepickerWithEvents {
 class DatepickerOpeningOnFocus {
   @ViewChild(MatDatepicker) datepicker: MatDatepicker<Date>;
 }
+
+
+@Component({
+  template: `
+    <input [matDatepicker]="ch">
+    <mat-datepicker #ch [calendarHeaderComponent]="CustomHeaderForDatepicker"></mat-datepicker>
+  `,
+})
+class DatepickerWithCustomHeader {
+  @ViewChild('ch') datepicker: MatDatepicker<Date>;
```

Comment:
Don't need these `ViewChild` since they aren't used for anything in the test

Reply:
I left the `datepicker` in https://github.com/angular/material2/pull/10856/commits/47acc75b84f8afe9142c9a043893ddae7aac4b38 for the moment